backend/src/main/resources/doc/openapi.json

```diff
@@ -1191,6 +1194,34 @@
                 "required": [
                     "type"
                 ]
+            },
+            "Badge": {
+                "type": "object",
+                "title": "Badge",
+                "description": "Badge owned by the user",
+                "properties": {
+                    "type": {
+                        "type": "string"
+                    },
+                    "level": {
+                        "type": "integer"
+                    },
+                    "updatedOnDate": {
```

Comment:
I don't remember. What's the purpose of this date?

Reply:
I think we need for downgrading, since we'll run scheduler once a day and check if the user missed another check up related to this badge type within predefined interval and if so, we'll downgrade the badge level

Reply:
I thought that we would be checking the check-up dates. Maybe I missed something so I need an explanation please. Let's write on Slack.